internal/middleware/objects/permissions.go

```diff
@@ -37,7 +38,26 @@ func AddFilePermissions(ctx context.Context) error {
 				Relation:    "parent", // this will always be parent in an object owned permission setup
 			})
 
-			if _, err := utils.AuthzClientFromContext(ctx).WriteTupleKeys(ctx, []fgax.TupleKey{req}, nil); err != nil {
+			tuples := []fgax.TupleKey{req}
+
+			// if the parent of the file is a user, explicitly add view permissions for org members
+			if f.Parent.Type == "User" {
+				orgID, err := auth.GetOrganizationIDFromContext(ctx)
+				if err != nil {
+					return err
+				}
+
+				orgReq := fgax.GetTupleKey(fgax.TupleRequest{
+					SubjectID:   orgID + "#member",
+					SubjectType: "organization",
+					ObjectID:    f.ID,
+					ObjectType:  "file",
+					Relation:    "can_view",
+				})
```

Comment:
I realize the lines a few above aren't doing this, but we should use the defined constants, also should use the `SubjectRelation` field instead:
```suggestion
				orgReq := fgax.GetTupleKey(fgax.TupleRequest{
					SubjectID:   orgID,
					SubjectType: generated.TypeOrganization,
				        SubjectRelation: fgax.MemberRelation,
					ObjectID:    f.ID,
					ObjectType:  generated.TypeFile,
					Relation:    fgax.CanView
				})
```

Reply:
resolved in https://github.com/theopenlane/core/pull/810/commits/dc66ffcc3eef0b1577beebb9b55ccdedcb7eac29